dashel/dashel-win32.cpp

```diff
@@ -1480,7 +1483,7 @@ namespace Dashel
 			unlock();
 
 			// force finite timeout to check for serial disconnections
-			DWORD r = WaitForMultipleObjects(hc, hEvs, FALSE, ms == INFINITE ? 1000 : ms);
+			DWORD r = WaitForMultipleObjects(hc, hEvs.data(), FALSE, ms == INFINITE ? DEFAULT_WAIT_TIMEOUT : ms);
```

Comment:
I think that the 1000 value was added to let the Dashel Win32 hub to check periodically for device disconnection. How large is `DEFAULT_WAIT_TIMEOUT`?

Reply:
1000, i extracted it to a constant, it could be shorter though

Reply:
ok, good!